CalculatorExercises/1-CalculatorBasic/Calculator.java

```diff
@@ -0,0 +1,43 @@
+import java.util.*;
+import java.lang.*;
+
+public class Calculator {
+   
+    private double result;
+
+    public double getResult() {
+        return result;
+    }
+
+    public Calculator () {
+        this.result = 0;
+    }
+
+    public void calculate (String input) {
+        try {
+                switch (input.charAt(0)) {
+                case '+':
+                    result += Double.valueOf(input.substring(1)); 
+                    break;
+                case '-':
+                    result -= Double.valueOf(input.substring(1));
+                    break;
+                case '/':
+                    result /= Double.valueOf(input.substring(1));
+                    break;
+                case '*':
+                    result *= Double.valueOf(input.substring(1));
+                    break;
+                default : 
+                    System.out.println("Zle polecenie");
+                    break;
+                }       
+            
+            } catch (StringIndexOutOfBoundsException err) {
```

Comment:
You are writing same output to user for every exception so you can catch 
catch (Exception exception) {...
you can do it because every exception is based on Exception 

If you want to catch every exception separately then
write some more information like:
} catch (StringIndexOutOfBoundsException exception) {
            System.out.println("Zle polecenie, polecenie za krotkie");
        } catch (NumberFormatException exception) {
            System.out.println("Zle polecenie, podano niepoprawna liczbe");
        }

Reply:
also wrong formatting (intendation)